config/site_settings.yml

```diff
@@ -1491,7 +1491,7 @@ files:
     max: 1024000
     type: file_size_restriction
   system_user_max_attachment_size_kb:
-    default: 0
+    default: 4096
```

Comment:
In my mind, the benefit of setting the default to `0` meant the setting was `disabled` and then we could just lean on the standard `max_attachment_size_kb` setting for size... Can you provide your thoughts on why you went with `4096` as a default @rngus2344 / @Grubba27

Reply:
I've opted to add it because it looked "right" given other [settings](https://github.com/discourse/discourse/blob/65b13a803dd6136aa428151beb8aad50d0a29ffa/config/site_settings.yml#L1483-L1497), but as we do [check now if the setting is zero(disabled)](https://github.com/discourse/discourse/blob/65b13a803dd6136aa428151beb8aad50d0a29ffa/app/controllers/uploads_controller.rb#L319-L325), I think it makes sense.

I'll revert it